### DISCUSSION TITLE
Change NGINX proxy manager default ports

### DISCUSSION THREAD

**Discussion by notorioustubze** [tteck/Proxmox#715] [Help Needed]
Hi @tteck thank you for all the great work you do! Is it at all possible to change the listening ports for NGNIX from 80 and 443 to 8080 4443? 

TIA!

**Comment by tteck** [comment]
Have never tried. Take a look at `nano /etc/nginx/conf.d/default.conf`

**Comment by golfhou** [comment]
Did you figure this out? 

I had NPM running on Unraid with different ports, and my router had those ports forwarding.  I am now trying to setup with Proxmox but, 443/80 are not coming through to my pfsense (zero activity), so maybe the modem is blocking these ports?

I updated the above file to use the same ports as on my unraid (under listening), but it still does not work.

**Reply by golfhou** [reply]
I modified the default.conf and was finally able to get NPM to connect through the modified ports, however, I was getting "Invalid SSL certificate Error code 526".  I lowered the the cloudflare encryption mode, and issued new origin server certificates, and got same results.

There must be another file that needs to be edited with different HTTPS ports in order for it to work.

Finally went back to 80/443, and opened/forwarded ports on pfsense to get it working normally.  Would like option to change ports.

BTW, also ran a quick docker compose with NPM using my original ports and that worked. So another option.
